.spelling

```diff
@@ -41,6 +41,7 @@ Databricks
 DataDog
 DataFrame
 DataFrames
+datapipeline
```

Comment:
drive-by comment: can you surround this and other additions to `.spelling` with backticks in the main text rather than adding them here? Including them as `#datapipeline` will avoid skip spell checking (similar for other slack channels / host names).